src/Midgard/CreatePHP/Mapper/AbstractRdfMapper.php

```diff
@@ -145,14 +156,17 @@ protected function getField($object, RdfElementDefinitionInterface $child)
     }
 
     /**
-     * A dummy classname canonicalizer: returns the name unmodified.
+     * {@inheritDoc}
      *
-     * @param string $className
-     * @return string exactly the same as $className
+     * The default implementation uses get_class on objects
      */
-    public function canonicalName($className)
```

Comment:
lets just `@deprecate` this method and have objectToName still call canonicalName so that we keep BC.